package.json

```diff
@@ -26,6 +26,7 @@
       "test"
     ],
     "testRegex": "test/(.*?/)?.*test.js$",
+    "testURL": "http://localhost",
```

Comment:
what's this config used for?

Reply:
On osx last versions of jest throws error, `localStorage is not available for opaque origins`. It solves only with this option in config.